data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,329 @@
+		to spawn
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic disables
+		ship "Lightning Bug (Ionic)" "Erazer"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		dialog "You receive a message from the remaining Unfettered ships: they surrender."
+		on disable
+			set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			has "old reputation: Hai (Unfettered)" < -2000
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained entering plunders staying target heroic
+		ship "Sea Scorpion (Shredder)" "Karma"
+		system "Hi Yahr"
+		on disable
+			set "unfettred: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		ship "Lightning Bug (Unfettered Shipyards)"
+		personality launching staying surveillance
+		on kill
+			fail "Hai Legacy 3"
+	on visit
+		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have more than just the ladybug."
+	to complete
+		has "unfettered: beaten the bugs"
+		or
+			"old reputation: Hai (Unfettered)" > -2000
+			has "unfettered: beaten extra bugs"
+	on complete
+		dialog `The engineer seems shocked at first. "You really... won? I did not participate in the fight, as you may have noticed. But I only have one word and will give you the license as agreed, but make sure you do not lose this until you may land on our worlds by other means, or we will take it very seriously. Oh, and we added taxes to the daily fee, I hope you don't mind."`
+		outfit "Unfettered Landing License"
+		log "Managed to use an old ladybug to overtake a swarm of Lightning Bugs, earning myself a place in Unfettered territory."
+
+mission "Unfettered Landing License Lost"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Hai Legacy 3: done"
+		not "outfit (all): Unfettered Landing License"
+		not "Unfettered Jump Drive 1: offered"
+	on offer
+		dialog "As you land the Unfettered immediately notice you do not have your license anymore. They warn you that this is your last chance, but you may have another one if you accept a fine of 2.5 million credits."
+	on accept
+		fail "Unfettered Landing License Lost"
+		outfit "Unfettered Landing License"
+		fine 2500000
+		log "Lost the landing license that the Unfettered provided but they were kind enough to provide another one for a loss fine of 2.5 million credits."
+	on decline
+		"reputation: Hai (Unfettered)" <?= -1
+		log "Lost the landing license that the Unfettered provided."
+
+mission "License Irrelevant"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+	on offer
+		outfit "Unfettered Landing License" -1
+		# it will now become free of upkeep
+		set "license: Unfettered Landing"
```

Comment:
Does this make a difference?

Reply:
If this doesn't work, maybe you could have a different outfit with the same name (`display name`), free-of-charge?

Reply:
I differentiated it fully and now give the license anyway when you give a JD